docs/server.rst

```diff
@@ -36,12 +36,54 @@ Prerequisites
 -------------
 
 The first thing you need is a recent (>= 2.6.36) Linux kernel that supports L2TPv3
-tunnels. The following modules are required for Tunneldigger operation:
+tunnels.
+
+These distributions are supported:
+
+* Debian
+* Fedora with the package ``kernel-modules-extra``
+* *add your distribution*
+
+Kernel Modules
+~~~~~~~~~~~~~~
+
+The following modules are required for Tunneldigger operation:
 
 * ``l2tp_core``
 * ``l2tp_eth``
 * ``l2tp_netlink``
 
+You can find out if these modules are loaded by running ``lsmod | grep l2tp``.
+If you get no output, they need to be activated.
+For the activation of the kernel modules, we recommend adding a file
+``/etc/modules-load.d/tunneldigger.conf`` with the following content:
+
+.. code:: shell
+
+    l2tp_core
+    l2tp_eth
+    l2tp_netlink
+
+You can also activate the modules using modprobe.
+A restart will load the kernel without them if you do not create the file.
+
+.. code:: shell
+
+    $ sudo modprobe l2tp_core
+    $ sudo modprobe l2tp_eth
+    $ sudo modprobe l2tp_netlink
+
+Then, your listing of the modules can look like this:
```

Comment:
```suggestion
If the modules were loaded successfully, your listing of the modules can look like this:
```

Reply:
Wow, how do you add this suggestion?

Reply:
Magic. ;)

When adding in-line comments in a PR, there's a button above the text box that looks like a document icon with "+ -" in it. That lets you write in-line suggestions that you can accept directly in the web UI.